src/foam/nanos/pm/PM.js

```diff
@@ -82,6 +106,21 @@ foam.CLASS({
       javaCode: `
     fm.foldForState(getClassType().getId()+":"+getName(), getStartTime(), getTime());
       `
+    },
+    {
+      name: 'error',
+      args: [
+        { name: 'x', type: 'Context' },
+        { name: 'exception', type: 'Object' },
+        { name: 'message', type: 'String...' }
```

Comment:
Can this be Object, then walk the list looking for an Exception.  Then exception can be anywhere.